src/assets/scss/material.orcid.overwrites.scss/button.scss

```diff
@@ -21,10 +21,41 @@ button.mat-button-wrap-text-with-icon {
   border: solid 1px rgba(0, 0, 0, 0.12);
   margin-top: 12px;
 
-  mat-icon {
+  a {
     position: absolute;
-    left: 5%;
-    top: 20%;
+    left: 20px;
+    top: 5px;
```

Comment:
![image](https://user-images.githubusercontent.com/2119626/82932788-31305680-9f46-11ea-84d6-c1bd12be8d99.png)

This does not look in the middle on mobile

Reply:
Good catch!. I will fix it.